InfinityCosmos/ForMathlib/AlgebraicTopology/SimplicialSet/Monoidal.lean

```diff
@@ -0,0 +1,6 @@
+import Mathlib.AlgebraicTopology.SimplicialSet.Monoidal
+import InfinityCosmos.ForMathlib.AlgebraicTopology.SimplexCategory
+
```

Comment:
Ditto add an extra line here and then add a comment like in the other file.

Reply:
More to the point, this file does not compile for me. You can check whether your branch compiles by opening up a terminal and typing "lake build".

Once you make the name changes above, you can fix the build by adding
```
universe u 

open CategoryTheory Limits Simplicial
```
to the top of the Monoidal file (after the imports).